staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/status_strategy.go

```diff
@@ -67,13 +67,6 @@ func (a statusStrategy) PrepareForUpdate(ctx context.Context, obj, old runtime.O
 	// track changed fields in the status update.
 	managedFields := newCustomResourceObject.GetManagedFields()
 
-	// KCP PATCH START
```

Comment:
The fact this commit is here means we added in some before, and we need to clean and remove them. 
This should not exist anymore.

Reply:
Seems to have been added in https://github.com/kcp-dev/kubernetes/commit/c6b52efab7ee329a8a20b51de5640c93685b2ac3 . I'll try to go through it and see if there are any more leftovers from this.

Reply:
Squashing this commit, [`5c809ee` UPSTREAM: \<squash\>: remove syncer custom code from apiextensions-apiserver](https://github.com/kcp-dev/kubernetes/commit/5c809ee29d646d084bc603570887e8fc42067676) into its origin, where this code comes from -- [`879e48` UPSTREAM: \<carry\>: apiextensions-apiserver](https://github.com/kcp-dev/kubernetes/commit/879e485b01a5137e8055bc333710f9cc460b931d).